configure.js

```diff
@@ -1,10 +1,17 @@
-var ipc = require('ipc')
 var Ractive = require('ractive')
 var page = require('page')
 var fs = require('fs')
+var Client = require('electron-rpc/client')
+var client = new Client()
```

Comment:
oh wait lol, i should read the full PR first, didn't realize it was a third party module